### PR TITLE
Smedberg/174192051 readme

### PR DESCRIPTION
**Pull request by smedberg**
Updating README.md to describe how to set up a dev environment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,42 @@
...
+This is a Rails application which demonstrates a system for allowing claimants to initiate and manage unemployment claims, and for customer service representatives to process claims.
+
+# Prerequisites
+
+* [rvm](https://rvm.io/)
+* Ruby 2.7.1 (`rvm install "ruby-2.7.1"`)
+* Node and NPM (`sudo apt update && sudo apt install nodejs npm`)
+* Webpack (`npm install --save-dev webpack && npm install --save-dev webpack-cli`)
+* Postgres (`sudo apt update && sudo apt install postgresql postgresql-contrib` on Ubuntu)
+* Postgres user `papua/papua` (`sudo -u postgres psql -c "CREATE USER papua WITH PASSWORD 'papua';"`)
```
Comment (MatthewSchultz) [inline README.md:12]: There are a buttload of ways to configure Postgress....not sure we want to specify one of them - or at least we probably want to say "well, you can do this how you like, but if you don't know how you like, here's a way to do it:"

File: README.md [hunk 2]
```diff
@@ -1,24 +1,42 @@
...
+* Edit `config/database.yml` to add info to `default` section:
+	* `username: papua`
+	* `password: papua`
+	* `host: localhost`
+* Create the database: `rake db:create`
+* Run database migrations: `rails db:migrate RAILS_ENV=development`
+* Set up Rails credentials for Alloy:
+	* Request a copy of the `master.key` file from another team member
+	* Store `master.key` in the `config` directory- it should be in the same folder as `credentials.yml.enc`
+* Start the rails server `rails server`
```
Comment (MatthewSchultz) [inline README.md:31]: In general, we prefer starting with foreman

File: README.md [hunk 3]
```diff
@@ -1,24 +1,42 @@
...
+* Change to the repo directory: `cd papua-id-demo`
+* Install Ruby dependencies: `bundle install`
+* Install Yarn dependencies: `yarn install --check-files`
+* Create a `config/database.yml` file (`cp config/database.yml.example config/database.yml`)
+* Edit `config/database.yml` to add info to `default` section:
+	* `username: papua`
+	* `password: papua`
+	* `host: localhost`
+* Create the database: `rake db:create`
+* Run database migrations: `rails db:migrate RAILS_ENV=development`
```
Comment (MatthewSchultz) [inline README.md:27]: Don't need to specify environment here, and not doing so will make the test database work too. Not that we actually HAVE any tests, but....

File: README.md [hunk 4]
```diff
@@ -1,24 +1,42 @@
...
+* Set up Rails credentials for Alloy:
```
Comment (MatthewSchultz) [inline README.md:28]: Not just alloy, in theory, there's other stuff in there.

File: README.md [hunk 5]
```diff
@@ -1,24 +1,42 @@
...
+	* Request a copy of the `master.key` file from another team member
+	* Store `master.key` in the `config` directory- it should be in the same folder as `credentials.yml.enc`
+* Start the rails server `rails server`
+* Visit [http://localhost:3000/](http://localhost:3000/) to see the server running!
+
+# Testing
```
Comment (MatthewSchultz) [inline README.md:34]: Tests don't work. At all. Testing section should basically say: "Testing: Don't do it - there are no automated tests. This is a demo, not a real app. Test it by running it and trying to figure out how to break it (not hard)."

**Review by MatthewSchultz — CHANGES_REQUESTED** [review]

File: README.md [hunk 6]
```diff
@@ -1,24 +1,42 @@
...
+* Set up Rails credentials for Alloy:
+	* Request a copy of the `master.key` file from another team member
+	* Store `master.key` in the `config` directory- it should be in the same folder as `credentials.yml.enc`
+* Start the rails server: `rails server`
+* Visit [http://localhost:3000/](http://localhost:3000/) to see the server running!
+
+# Testing
+
+* Set up Rails credentials for Alloy:
+	* `rails credentials:edit --environment test`
```
Comment (smedberg) [inline README.md:37]: This is wrong - will remove (all testing stuff is wrong anyway :^P )

File: README.md (hunk 1 above)
Comment (smedberg) [reply README.md:12]: Will generalize, offer this as "one way to do it"

File: README.md (hunk 3 above)
Comment (smedberg) [reply README.md:27]: Will update

File: README.md (hunk 4 above)
Comment (smedberg) [reply README.md:28]: Will generalize

File: README.md (hunk 5 above)
Comment (smedberg) [reply README.md:34]: Will update

**Review by MatthewSchultz — APPROVED** [review]

**Pull request merged**